persistent/test/Database/Persist/TH/EntityHaddockSpec.hs

```diff
@@ -0,0 +1,36 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE TemplateHaskell #-}
+
+module Database.Persist.TH.EntityHaddockSpec (spec) where
+
+import TemplateTestImports
+
+#if MIN_VERSION_template_haskell(2,18,0)
+import Database.Persist.TH.CommentSpec (CommentModel (..))
+import Language.Haskell.TH (DocLoc (DeclDoc), getDoc)
+import Language.Haskell.TH.Lift (lift)
```

Comment:
```suggestion
import Language.Haskell.TH.Syntax (lift)
```

Should work to allow removing the `th-lift` dependency

Reply:
Fixed in 3fbb38362faba6465d4894bc64ae0ec907fc1ee8.